src/index.html

```diff
@@ -16,17 +16,128 @@ <h1>Recipe Easy</h1>
      </header> <!--END OF NAV BAR-->
 
 
-     <main> <!--START OF MAIN SECTION-->
+<!--      <main> START OF MAIN SECTION
         <h1>Categories</h1>
+      </main>
 
-        <h1>Meals</h1>
-     </main> <!--END OF MAIN SECTION-->
+ -->      <section class="meals">
+         <h1 class="title">MEALS</h1>
+ 
+         <article class="meals-card">
+             <div class="meals-content">
```

Comment:
nit: perhaps the article could have the class name of meals-content (since it has all the cards) and each div inside the class name of meals-card